addons/analytic/tests/test_analytic_account.py

```diff
@@ -71,6 +71,14 @@ def setUpClass(cls):
             'analytic_distribution': {cls.analytic_account_2.id: 100}
         })
 
+        """ Removes access rights linked to timesheet and project as these add
+        record rules blocking analytic flows; account overrides it"""
+        if 'account.account' not in cls.env:
+            core_group_ids = (cls.env.ref("project.group_project_manager", raise_if_not_found=False) or cls.env['ir.rule']) | (cls.env.ref("hr_timesheet.group_hr_timesheet_user", raise_if_not_found=False) or cls.env['ir.rule'])
```

Comment:
I think we can just remove `hr_timesheet.group_hr_timesheet_user`, I guess `project.group_project_manager` will automatically be removed since project manager implies `hr_timesheet.group_hr_timesheet_approver`, no? 🤔 

Apart of that, LGTM. 🙂